app/public/index.html

```diff
@@ -1,14 +1,107 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title></title>
-</head>
-<body>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Login Page</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.3.1/css/all.css"
+      integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU"
+      crossorigin="anonymous" />
+  </head>
+  <body>
     <!-- Richard: This will send a message to the websocket server -->
-    <button id="send">Send message to client</button>
-</body>
-<script src="./constants.js"></script>
-<script src="./script.js"></script>
+    <div class="login-container">
+      <h2>Login</h2>
+      <form id="loginForm">
+        <div class="input-group">
+          <label for="username">Username</label>
+          <input
+            type="text"
+            id="username"
+            name="username"
+            placeholder="Username"
+            required />
+        </div>
+        <div class="input-group">
+          <label for="password">Password</label>
+          <input
+            type="password"
+            id="password"
+            name="password"
+            placeholder="Password"
+            required />
+        </div>
+        <button type="submit" id="submit">Login</button>
+        <div id="error-message" style="color: red"></div>
+      </form>
+    </div>
+
+    <!-- <button id="send">Send message to client</button> -->
+    <div class="create-container">
+      <h2>Create Account</h2>
+      <form id="createAccountForm">
+        <div class="input-group">
+          <label for="newUsername">Username</label>
+          <input
+            type="text"
+            id="newUsername"
+            name="newUsername"
+            placeholder="Username"
+            required />
+        </div>
+        <div class="input-group">
+          <label for="newPassword">Password</label>
+          <input
+            type="password"
+            id="newPassword"
+            name="newPassword"
+            placeholder="Password"
+            required />
+        </div>
+        <div class="input-group">
+          <label for="newPasswordConfirm">Confirm Password</label>
+          <input
+            type="password"
+            id="newPasswordConfirm"
+            name="newPasswordConfirm"
+            placeholder="Confirm Password"
+            required />
+        </div>
+
+        <button type="submit" id="createAccount">Create Account</button>
+      </form>
+
+      <div id="set">
+        <div id="count">Length : 0</div>
+        <i id="see" onclick="see()" class="far fa-eye"></i>
```

Comment:
The "see" function is undefined. I imagine you'll have this reveal the user's password instead of dots when this button is clicked. I recommend declaring an empty `see()` function for now so this stops throwing an error.